contentcuration/contentcuration/frontend/channelEdit/views/list/ContentNodeListItem.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+
+  <li
```

Comment:
Can also use [VListTile tag](https://codepen.io/pen/?&editable=true&editors=101) here to handle some of the styling (you may need to overwrite the max-height/height style rules, but the rest should be okay to use)

Reply:
Your link doesn't show anything

Reply:
oops! haha Here's the main page https://v15.vuetifyjs.com/en/components/lists#usage